codersdk/templates.go

```diff
@@ -30,6 +31,12 @@ type UpdateActiveTemplateVersion struct {
 	ID uuid.UUID `json:"id" validate:"required"`
 }
 
+type UpdateTemplateMeta struct {
+	Description                string `json:"description" validate:"omitempty"`
+	MaxTTLMillis               int64  `json:"max_ttl_ms" validate:"omitempty"`
+	MinAutostartIntervalMillis int64  `json:"min_autostart_interval_ms" validate:"omitempty"`
```

Comment:
```suggestion
	Description                string `json:"description,omitempty"`
	MaxTTLMillis               int64  `json:"max_ttl_ms,omitempty"`
	MinAutostartIntervalMillis int64  `json:"min_autostart_interval_ms,omitempty"`
```

I didn't see any uses of "validate omit" in the code base, so I think this was the intent?